smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -45,17 +45,33 @@ def setUp(self):
             for tmp in Section.interfaces("ethernet"):
                 if not '.' in tmp:
                     self._members.append(tmp)
+            
+            self.session.set(['interfaces','dummy','dum0'])
+            self.session.set(['interfaces','dummy','dum1'])
+            self.session.commit()
+            self.session.set(['interfaces','bonding','bond1','member','interface','dum0'])
+            self.session.set(['interfaces','bonding','bond1','member','interface','dum1'])
+            self.session.commit()
+            for tmp in Section.interfaces("bonding"):
+                if not '.' in tmp:
+                    self._members.append(tmp)
 
         self._options['br0'] = []
         for member in self._members:
             self._options['br0'].append(f'member interface {member}')
+    
+    def tearDown(self):
+        self.session.delete(['interfaces','bonding'])
+        self.session.delete(['interfaces','dummy'])
+        super().tearDown()
 
     def test_add_remove_member(self):
         """ Add member interfaces to bridge and set STP cost/priority """
         for interface in self._interfaces:
             base = self._base_path + [interface]
             self.session.set(base + ['stp'])
             self.session.set(base + ['address', '192.0.2.1/24'])
+            self.session.commit()
```

Comment:
Why is this `commit()` required?

Reply:
I remember that I have not modified irrelevant functions, maybe I added them inadvertently